hugo/content/snippets/javascript-loops-pro-tips.md

```diff
@@ -50,6 +51,7 @@ for (const v of arr) {}  // 11.7ms
 
 arr.forEach(v => v) // 2.1ms
 
+for (let i = mil; i > 0; i--) {} // ???
```

Comment:
@codediodeio sorry! should have been more explicit about this here! my times were:
```
⏲️: 3.208ms
⏲️: 20.858ms
⏲️: 2.357ms
⏲️: 1.861ms
```
I'm guessing on your's it'd be ~1.6? 1.7?

Reply:
Since this is merged, gimme a :+1: and I'll submit a new PR